### PR TITLE
Portworx chart update.

### PR DESCRIPTION
**Pull request by pault84**
We've released multiple new versions and the current version of the charts are not compatible with the newer version of px-enterprise/lighthouse.

This will address some required changes and version updates.

### CODE REVIEW

File: charts/portworx/v1.0.0/questions.yml [hunk 1]
```diff
@@ -137,7 +137,7 @@ questions:
   type: boolean
   group: "Advanced parameters"
 - variable: storkVersion
-  default: "1.2.0"
+  default: "2.0.1"
```
Comment (harsh-px) [inline charts/portworx/v1.0.0/questions.yml]: update to 2.1.0

File: charts/portworx/v1.0.0/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: portworx
-appVersion: 1.0.0
-version: 1.0.0
+appVersion: 1.0.1
+version: 1.0.1
```
Comment (harsh-px) [inline charts/portworx/v1.0.0/Chart.yaml]: the chart will need to move to  a new  v1.0.1 folder. Current changes are in v1.0.0 folder

File: charts/portworx/v1.0.0/questions.yml [hunk 3]
```diff
@@ -149,18 +149,31 @@ questions:
...
+  group: "Advanced parameters"
+
 - variable: envVars
   label: "Environment variables"
   description: "semi-colon-separated list of environment variables that will be exported to portworx. (example: API_SERVER=http://lighthouse-new.portworx.com;MYENV1=val1;MYENV2=val2)"
   type: string
   group: "Advanced parameters"
 - variable: imageVersion
-  default: "1.5.1"
+  default: "2.0.2"
```
Comment (harsh-px) [inline charts/portworx/v1.0.0/questions.yml]: change to 2.0.3.2

File: charts/portworx/v1.0.0/questions.yml [hunk 4]
```diff
@@ -149,18 +149,31 @@ questions:
   type: boolean
   group: "Advanced parameters"
 - variable: lighthouseVersion
-  default: "1.4.0"
+  default: "2.0.1"
```
Comment (harsh-px) [inline charts/portworx/v1.0.0/questions.yml]: update to 2.0.3

File: proposed/longhorn/latest/Chart.yaml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: longhorn
-version: 0.4.0
-appVersion: v0.4.0
-kubeVersion: ^1.8.x
+version: 0.4.1
+appVersion: v0.4.1
```
Comment (harsh-px) [inline proposed/longhorn/latest/Chart.yaml]: the longhorn changes should not be in this PR. Rebase with latest master.

**Comment by harsh-px** [comment]
The helm lint is expected to fail as installing the Portworx helm chart with default parameters will always fail. Users need to provide a a kvdb argument.

For e.g 

`helm install --dry-run --debug --set kvdb=etcd:http://192.168.70.90:2379,clusterName=$(uuid) .`

**Comment by harsh-px** [comment]
Could we get some reviews here?

**Comment by guangbochen** [comment]
LGTM.

**Comment by harsh-px** [comment]
@guangbochen thanks for the approval.

What is the release process for this chart to be available for all rancher users? I see the current merge branch as `dev`. I'm guessing you guys would be merging this from dev to master and then releasing it from there.

**Comment by guangbochen** [comment]
yes, the QA will test the updated charts and we will merge those charts into master after that, thanks.

**Pull request merged**